pages/dkp/kommander/2.1/release-notes/2.1.0/index.md

```diff
@@ -202,6 +202,22 @@ flux reconcile -n kommander-flux source git management --kubeconfig MANAGED_KUBE
 ✔ fetched revision main/GIT_HASH
 ```
 
+### Cert-manager expiration workaround
+
+Due to an oversight, some Kommander versions do not properly handle certificate renewal for Kommander applications. The`cert-manager` component renews all certificates 60 days after you install Kommander on your cluster. When this occurs, some Kommander applications and pods fail to receive the renewed certificate information, causing them to stop working upon expiration. This occurs 90 days after Kommander was installed, which normally would coincide with the date you created the cluster. While the effects can vary, the most common failure is the inability to log in to the UI due to an expired certificate in the dex-k8s-authenticator pod.
+
+A permanent fix for the issue requires upgrading to Kommander 2.2.1 or higher. In the meantime, there is a workaround available that forces the applications to reconcile and recognize the renewed certificate. This workaround also extends the validity of the certificates to 10 years, fixes the certification reload issue, and restarts the affected pods once the new certificate is issued.
+
+The workaround applies to any environment (networked, air-gapped, on-prem, etc.) and fixes the issue regardless of your issuer type ([SelfSigned][selfsigned] for air-gapped environments, [ACME][acme], or your own certificate issuer configured separately for your institution).
```

Comment:
```suggestion
The docker container can be applied to the management cluster in any environment (networked, air-gapped, on-prem, etc.) and remediate the issue regardless of your CA issuer type ([SelfSigned][selfsigned] for air-gapped environments, [ACME][acme], or your own certificate issuer configured separately for your institution).   

If there are any workload clusters attached to the management cluster, they will also be remediated.

Note that this container does affect or change in any way certificates associated with a custom domain you may have created for the cluster.  
```

Reply:
@cneth just double-checking. I think there is a "not" missing in line 215? 

```suggestion
Note that this container does not affect or change in any way certificates associated with a custom domain you may have created for the cluster.  
```

Reply:
Made an edit - Docker should be caps'ed